File: hrm-app-api/HRM_Api/DTOs/EmployeefamilyInfoDto.cs
namespace HRM_Api.DTOs
{
    public class EmployeefamilyInfoDto
    {
        public int Id { get; set; }
        public int IdClient { get; set; }
        public string Name { get; set; } = null!;
        public int IdGender { get; set; }
        public int IdRelationship { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string? ContactNo { get; set; }
        public string? CurrentAddress { get; set; }
        public string? PermanentAddress { get; set; }
        public DateTime? SetDate { get; set; }
    }
}

File: hrm-app-client/src/app/models/employeefamilyInfoDto.ts
export interface EmployeefamilyInfoDto {
  id: number;
  idClient: number;
  name: string;
  idGender: number;
  genderName?: string;
  idRelationship: number;
  relationshipName?: string;
  dateOfBirth?: Date | null;
  contactNo?: string;
  currentAddress?: string;
  permanentAddress?: string;
  setDate?: Date | null;
}